getdata_HN.py

```diff
@@ -38,27 +38,32 @@ def _get_stories_of_topics(topics=None):
 
 
 def _save_stories(stories):
-    with open(DATAFILE, 'w') as outfile:
+    with open(FILE, 'w') as outfile:
         json.dump(stories, outfile, indent=4)
     return
 
 
 def main():
-    if os.path.isfile(DATAFILE):
+    topics = ['python', 'django']
+    if os.path.isfile(FILE):
         old_stories = dict()
         new_stories = dict()
-        stories = _get_stories_of_topics()
-        with open(DATAFILE) as json_data:
-            old_stories = json.load(json_data)
+        stories = _get_stories_of_topics(topics)
+        with open(FILE) as json_data:
+            try:
+                old_stories = json.load(json_data)
+            except IOError as err:
+                old_stories = {}
+                logging.error(err)
         if stories:
             old_stories.update(stories)
             new_stories = old_stories
         _save_stories(new_stories)
     else:
-        _save_stories(_get_stories_of_topics())
+        _save_stories(_get_stories_of_topics(topics))
 
 
 if __name__ == "__main__":
     start_time = time.time()
     main()
-    print("--- %s seconds ---" % (time.time() - start_time))
+    logging.info("--- %s seconds ---" % (time.time() - start_time))
```

Comment:
this will not printout anything. Default logging level is logging.WARNING. You must set logging level